### DISCUSSION TITLE
[Support]: Android Chrome PWA Issues

### DISCUSSION THREAD

**Discussion by Go2Engle** [blakeblackshear/frigate#12987] [General Support] [answered]
### Describe the problem you are having

Im using a galaxy s24 ultra with google chrome. When I browse the the frigate UI via IP and port and try to add to the home screen it only allows for the shortcut to be added and the option to install is not presented.

### Steps to reproduce

1. Open chrome
2. Tap three dots in top right
3. Tap Add to Home screen
4. Only "Displays Create shortcut"

### Version

0.14.0-da913d8

### Frigate config file

```yaml
n/a
```

### Relevant log output

```shell
n/a
```

### Operating system

HassOS

### Install method

HassOS Addon

### Network connection

Wireless

### Camera make and model

Mixed

**Comment by NickM-27** [comment, marked as answer]
I've been using pwa on my galaxy for months. You're most likely missing one of the requirements for PWA like not using a secure context

**Reply by Go2Engle** [reply]
ahhhh forgot about that! tossed it in my nginx proxy with an ssl cert and bam Install popped right up!
